chapters/annotations.tex

```diff
@@ -55,10 +55,13 @@ \section{Annotations for Documentation}\doublelabel{annotations-for-documentatio
 \lstinline!<html>! and \lstinline!</html>! define
 optionally begin and end of content that is HTML encoded. For external
 links see \autoref{external-resources}. Links to Modelica classes may be defined with
-the HTML link command using scheme \lstinline!Modelica! (the scheme in the URI is case-insensitive), e.g.,
+the HTML link command using scheme \lstinline!Modelica!, e.g.,
 \begin{lstlisting}[language=modelica]
 <a href="modelica://MultiBody.Tutorial">MultiBody.Tutorial</a>
 \end{lstlisting}
+\begin{nonnormative}
+The name of the scheme is case-insensitive, but the lower-case variant (in this case "modelica") is recommended.
+\end{nonnormative}
```

Comment:
This still looks to me as if we are giving the same rule in two different places.  I suggest that we make the recommendation in `autoref{external-resources}` normative, and add a reference to this rule in the current context instead of repeating the rule:
> Using scheme \lstinline!Modelica! (using its lower case form in the URI, see \autoref{external-resources}), e.g.,

Reply:
Ok.